Example/Podfile.lock

```diff
@@ -2,12 +2,12 @@ PODS:
   - CGFloatLiteral (0.2.0)
   - ManualLayout (1.3.0)
   - ReusableKit (1.1.0)
-  - RxCocoa (3.1.0):
-    - RxSwift (~> 3.1)
-  - RxKeyboard (0.4.0):
-    - RxCocoa (>= 3.0)
-    - RxSwift (>= 3.0)
-  - RxSwift (3.1.0)
+  - RxCocoa (3.5.0):
```

Comment:
Uhm very weird it would require 3.4 , but even if so let's try keep it to the minimum required? 3.5 isn't a requirement here. So besides that I have no problems with this PR